NextCenturyCorporation/neon-dash
# AIDA-214

created script that builds VERDI-Dash and deploys it on AWS Machine
To test:
- observe 34.201.36.109/aida
        - The highlighting should still be yellow; favicon should still be default neon

- run script (VERDI-Dash/scripts/deploy_verdi_dash.sh)
- observe 34.201.36.109/aida
        -The highlighting should now be light blue; favicon should be updated to the AIDA version
